src/services/account/services/jwtTimerService.js

```diff
@@ -0,0 +1,75 @@
+const auth = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+const { MethodNotAllowed } = require('@feathersjs/errors');
+
+const { getRedisClient, redisSetAsync, redisTtlAsync, getRedisIdentifier } = require('../../../utils/redis');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace. (object-curly-newline)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4141092)